functions/Get-SqlMaxMemory.ps1

```diff
@@ -86,51 +72,22 @@ Find all servers in CMS that have Max SQL memory set to higher than the total me
 				continue
 			}
 			
-			$maxmem = $server.Configuration.MaxServerMemory.ConfigValue
-			
-			$reserve = 1
-			$totalMemory = $server.PhysicalMemory
-			
-			
-			# Some servers underreport by 1MB.
-			if (($totalmemory % 1024) -ne 0) { $totalMemory = $totalMemory + 1 }
-			
-			if ($totalMemory -ge 4096)
-			{
-				$currentCount = $totalMemory
-				while ($currentCount/4096 -gt 0)
-				{
-					if ($currentCount -gt 16384)
-					{
-						$reserve += 1
-						$currentCount += -8192
-					}
-					else
-					{
-						$reserve += 1
-						$currentCount += -4096
-					}
-				}
-				$recommendedMax = [int]($totalMemory - ($reserve * 1024))
-			}
-			else
-			{
-				$recommendedMax = $totalMemory * .5
-			}
-			
-			$recommendedMax = $recommendedMax/$sqlcount
+
+			$totalmemory = $server.PhysicalMemory
 			
-			$object = New-Object PSObject -Property @{
+			# Some servers under-report by 1MB.
+			if (($totalmemory % 1024) -ne 0) { $totalmemory = $totalmemory + 1 }
+
+			$object = New-Object -TypeName PSObject -Property @{
```

Comment:
This line should now just be 

[pscustomobject]@{

You can see similar output here: https://github.com/sqlcollaborative/dbatools/blob/master/functions/Get-DbaStartupParameter.ps1#L141